var/spack/repos/builtin/packages/py-numpy-quaternion/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyNumpyQuaternion(PythonPackage):
+    """This package creates a quaternion type in python, and further enables
+    numpy to create and manipulate arrays of quaternions. The usual algebraic
+    operations (addition and multiplication) are available, along with numerous
+    properties like norm and various types of distance measures between two
+    quaternions. There are also additional functions like “squad” and “slerp”
```

Comment:
```suggestion
    quaternions. There are also additional functions like "squad" and "slerp"
```
Python 2 doesn't support unicode properly